tests/unit/parsers/test.manifestjson.js

```diff
@@ -1922,12 +1919,13 @@ describe('ManifestJSONParser', () => {
 
         const keys = Object.keys(contentSecurityPolicy);
         for (let i = 0; i < keys.length; i++) {
-          expect(warningsV3[i].code).toEqual(messages.MANIFEST_CSP_UNSAFE_EVAL);
-          expect(warningsV3[i].message).toContain(
+          expect(warningsV3[i * 2].code).toEqual(messages.MANIFEST_CSP_UNSAFE_EVAL);
```

Comment:
maybe mention that we expect two codes for each key, and so first warning is `MANIFEST_CSP_UNSAFE_EVAL` and second warning is `MANIFEST_CSP`.